specifications/api/swagger.yaml

```diff
@@ -3452,6 +3482,24 @@ paths:
         - *parametersAddressState
       responses: *responsesListAddresses
 
+  /wallets/{walletId}/keys:
```

Comment:
It would be more coherent for API users if the endpoint for retrieving this key could be similar to the getWalletKey endpoint. So parameters related to derivation path and chain code are on the request path. Only the passphrase would be in the request body.

Reply:
I agree, I'd be better to have the index as a path parameter here :+1:

Reply:
done- please refer HEAD